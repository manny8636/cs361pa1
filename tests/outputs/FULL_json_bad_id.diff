--- outputs/FULL_json_bad_id.txt	2022-02-15 14:30:55.603076444 -0500
+++ expected/FULL_json_bad_id.txt	2021-01-25 15:27:24.000000000 -0500
@@ -0,0 +1,3 @@
+SYNTAX ERROR: '\t' is not a valid escape code
+SYNTAX ERROR: Could not process text beginning at '"Not \t valid": "value" }'
+Parsing ../data/json_bad_id.txt failed
