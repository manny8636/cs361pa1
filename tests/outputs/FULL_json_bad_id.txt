Parsing ../data/json_bad_id.txt failed
