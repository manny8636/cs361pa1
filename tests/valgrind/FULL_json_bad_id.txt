==372162== Memcheck, a memory error detector
==372162== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==372162== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==372162== Command: ../json -o ../data/json_bad_id.txt
==372162== 
==372162== 
==372162== HEAP SUMMARY:
==372162==     in use at exit: 2,064 bytes in 1 blocks
==372162==   total heap usage: 4 allocs, 3 frees, 10,848 bytes allocated
==372162== 
==372162== LEAK SUMMARY:
==372162==    definitely lost: 2,064 bytes in 1 blocks
==372162==    indirectly lost: 0 bytes in 0 blocks
==372162==      possibly lost: 0 bytes in 0 blocks
==372162==    still reachable: 0 bytes in 0 blocks
==372162==         suppressed: 0 bytes in 0 blocks
==372162== Rerun with --leak-check=full to see details of leaked memory
==372162== 
==372162== For lists of detected and suppressed errors, rerun with: -s
==372162== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
