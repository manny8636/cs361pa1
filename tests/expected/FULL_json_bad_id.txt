SYNTAX ERROR: '\t' is not a valid escape code
SYNTAX ERROR: Could not process text beginning at '"Not \t valid": "value" }'
Parsing ../data/json_bad_id.txt failed
